dockerfiles/tasks.py

```diff
@@ -157,3 +160,33 @@ def buildassets(c):
     """Build all assets for the application and push them to backend storage"""
     c.run(f'docker-compose -f {DOCKER_COMPOSE_ASSETS} run --rm assets bash -c "npm ci && node_modules/bower/bin/bower --allow-root update && npm run build"', pty=True)
     c.run(f'{DOCKER_COMPOSE_COMMAND} run --rm web python3 manage.py collectstatic --noinput', pty=True)
+
+
+@task(help={
+    'action': 'Action to realize on Transifex ("pull" or "push")',
+})
+def translations(c, action):
+
+    if action not in ('pull', 'push'):
+        print(f'Action passed ("{action}") not supported. Use "pull" or "push".')
+        sys.exit(1)
+
+    transifex_token = os.environ.get('TRANSIFEX_TOKEN', None)
+    if not transifex_token:
+        print('You need to export TRANSIFEX_TOKEN environment variable.')
+        sys.exit(1)
```

Comment:
This seems like it should be a warning or requirement at the top of the deploy prepare process. Otherwise, the missing variable isn't known until it's too late.

Reply:
Yeah. I put it here because as this step is optional and can be skipped, I didn't want to fail the release if there is no Transifex token specified. I think there is a tradeoff here. Do you prefer to fail the process at the beginning anyway?

Another option could be using a warning as you mentioned at the start and always skip this step if the token is not provided. However, I'm sure this way it will probably always be skipped :)

Reply:
BTW, I think this is fine here, because this command _requires_ this variable. However, we can add extra code to the deploy/release script to make it nicer --but we shouldn't remove this from here.

Reply:
> However, we can add extra code to the deploy/release script to make it nicer --but we shouldn't remove this from here

Oh yeah, I was thinking this option as well. This argument makes sense here, but probably eventually also some mechanism in the prepare task would be helpful to keep translations on track. Keeping this optional for now seems fine, and perhaps adding a requirement to the prepare task is what makes this step mandatory.